blockchain/state/statedb.go

```diff
@@ -471,6 +471,13 @@ func (self *StateDB) SetState(addr common.Address, key, value common.Hash) {
 	}
 }
 
+func (self *StateDB) SetStorage(addr common.Address, storage map[common.Hash]common.Hash) {
```

Comment:
Can you add some comments for this?

Reply:
Added comment in [this commit](https://github.com/klaytn/klaytn/pull/1134/commits/55b832f760f17c68c4f595c7d6feffbd7d7437ac).